README.md

```diff
@@ -34,78 +29,100 @@ The following Visual Studio Code settings are available for the ECL extension.
   // Run 'eclcc -syntax' on load.
   "ecl.syntaxCheckOnLoad": true
 
-// External folders used by IMPORT
+  // Additional folders to use when resolving IMPORT statements.
   "ecl.includeFolders": []
 
-  // Override eclcc auto detection
+  // Override eclcc auto detection.
   "ecl.eclccPath": ""
 
-  // Add '-legacy' arguement to eclcc.
+  // Add '-legacy' argument to eclcc.
   "ecl.legacyMode": false
 
-  // Open workunits in external browser.
+  // Open Workunits in external browser.
   "ecl.WUOpenExternal": true
 
-  // Automatically open WU in browser on creation.
+  // Automatically open Workunits on creation.
   "ecl.WUAutoOpen": false
 
-  // Debug logging.
+  // Debug level logging (requires restart).
   "ecl.debugLogging": false
   
 ```
 
-#### Launch Settings
+## Launch Settings
 
-Submitting or debugging ECL using VS-Code requires specifying the target environment within the VS Code `launch.json` (pressing `F5` will prompt you to auto create a skeleton file if none exists):
+Submitting ECL using VS-Code requires specifying the target environment within the VS Code `launch.json` (pressing `F5` will prompt you to auto create a skeleton file if none exists):
 
 ```javascript
+// Default ECL Launch Configuration
 {
-    "name": "localhost-hthor",
-    "type": "ecl",
-    "request": "launch",
-    "mode": "submit", // "submit" | "compile" | "publish"
-    "workspace": "${workspaceRoot}",
-    "program": "${file}",
-    "protocol": "http",
-    "serverAddress": "localhost",
-    "port": 8010,
-    "abortSubmitOnError": true,
-    "rejectUnauthorized": false,
-    "targetCluster": "hthor",
-    "eclccPath": "${config:ecl.eclccPath}",
-    "eclccArgs": [],
-    "includeFolders": "${config:ecl.includeFolders}",
-    "legacyMode": "${config:ecl.legacyMode}",
-    "resultLimit": 100,
-    "user": "",     //  Optional (vscode will prompt as needed)
-    "password": ""  //  Optional (vscode will prompt as needed)
+  "name": "play-hthor-submit",
+  "type": "ecl",
+  "request": "launch",
+  "mode": "submit",
+  "workspace": "${workspaceRoot}",
+  "program": "${file}",
+  "protocol": "https",
+  "serverAddress": "play.hpccsystems.com",
+  "port": 18010,
+  "rejectUnauthorized": false,
+  "targetCluster": "hthor",
+  "eclccPath": "${config:ecl.eclccPath}",
+  "eclccArgs": [],
+  "includeFolders": "${config:ecl.includeFolders}",
+  "legacyMode": "${config:ecl.legacyMode}",
+  "resultLimit": 100,
+  "user": "vscode_user",
+  "password": ""
 }
 ```
 
-## Building and Debugging the Extension
+## ECL Commands
+
+The following ECL specific commands are available.  Note:  These commands will **not** be live until an ECL file has been opened (as this triggers the extension to load).  To activate a command either use its associated hotkey or press `ctrl/cmd + shift + p` and type `ECL` this will present a filtered list of the ECL specific commands:
```

Comment:
These commands will **not** be live
s/b 
These commands will **not** be active